core/src/main/java/org/apache/calcite/avatica/util/AbstractCursor.java

```diff
@@ -908,11 +887,16 @@ private byte[] getBase64Decoded() throws SQLException {
    * Accessor that assumes that the underlying value is a DATE,
    * in its default representation {@code int};
    * corresponds to {@link java.sql.Types#DATE}.
+   *
+   * <p>For backwards compatibility, timezone offsets are calculated in relation to the local
+   * timezone instead of to UTC. Providing the default calendar will return the underlying value
+   * unmodified. This differs from {@link DateAccessor} which uses the timezone offset from UTC.
+   * </p>
```

Comment:
We generally don't use `</p>` when writing javadoc. Only `<p>`, after a blank line, as you have done already.